Things_You_Should_Know/things_you_should_know.md

```diff
@@ -0,0 +1,400 @@
+ 
+
+### App Settings are injected into your app as environment variables at runtime
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+If you need to set an environment variable for your application, simply add an App Setting in the Azure portal. When your app runs, we will inject the app setting into the process as an environment variable automatically.
+
+
+### Some characters may be stripped out of environment variables names
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+You can use only letters (A-Z, a-z), numbers (0-9), and the underscore character (_) for application settings.
+
+Any other characters will be stripped from the App Setting name (and, therefore, the environment variable name) at runtime. 
+
+
+### Your environment variables won't appear in the Bash console
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+If you use the Bash console in [**Kudu** (Advanced Tools)](https://github.com/projectkudu/kudu/wiki) to look at environment variables, you won't see environment variables that you set using App Settings. That's because the Bash console runs in a separate context from your App. In order to see your environment variables, use the **Environment** page in **Kudu** or use code to retrieve the environment variables.
+
+### Set your default document in a Node.js app using JavaScript
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+When you create a Node.js app, by default, it's going to use `hostingstart.html` as the default document unless you configure it to look for a different file. You can use a JavaScript file to configure your default document. 
+
+Create a file called index.js in the root folder of your app and add the following content.
+
+``` Node
+var express = require('express');
+var server = express();
+var options = {
+index: 'index.html'
+};
+server.use('/', express.static('/home/site/wwwroot', options));
+server.listen(process.env.PORT);
+```
+
+This will configure index.html as the default document for your app.
+
+
+### You can now force SSL using the "HTTPS Only" option in the Azure portal
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+**App Service** includes the ability to force SSL traffic using the "HTTPS Only" option in the Azure portal. For details on how to do that, see [Secure a custom DNS name with an SSL binding in Azure App Service](https://docs.microsoft.com/azure/app-service/configure-ssl-bindings#enforce-https).
+
+### If you're persisting files, only the /home directory is persisted
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+By default, we don't persist files in your app. However, if you need to persist files, you can enable **App Service** built in storage (see [You can enable and disable storage persistence with an app setting](You-can-enable-and-disable-storage-persistence-with-an-app-setting)). 
```

Comment:
This contradicts the earlier section about enabling persistent storage.  Also for bring your own code we certainly do persist files written to /home.